gaugan/callbacks.py

```diff
@@ -31,32 +34,50 @@ def infer(self):
         )
         return self.model.predict([latent_vector, self.val_images[2]])
 
+    def log_to_tables(self, epoch, semantic_input, ground_truth, generated_images):
+        wandb_table = wandb.Table(columns=self.columns)
+        for i in range(4):
```

Comment:
this 4 number could be explained or made a global variable. It's currently a bit of a magic number